docs/.generated_docs

```diff
@@ -208,7 +208,6 @@ docs/man/man1/kubectl-rollout-resume.1
 docs/man/man1/kubectl-rollout-status.1
 docs/man/man1/kubectl-rollout-undo.1
 docs/man/man1/kubectl-rollout.1
-docs/man/man1/kubectl-run-container.1
```

Comment:
@kubernetes/sig-cli-maintainers is it ok to remove those ? although `run-container` is a deprecated command [here](https://github.com/kubernetes/kubernetes/blob/81631eceb80aace00dfb221b13440c7f658a18c7/pkg/kubectl/cmd/cmd.go#L250)